stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -27,18 +30,20 @@ public AndroidDOMProvider(Application application) {
     mDescriptorMap = new DescriptorMap()
         .beginInit()
         .register(Activity.class, new ActivityDescriptor())
-        .register(Application.class, new ApplicationDescriptor())
+        .register(Application.class, new ApplicationDescriptor());
+    FragmentDescriptor.register(mDescriptorMap)
         .register(Object.class, new ObjectDescriptor())
         .register(TextView.class, new TextViewDescriptor())
         .register(View.class, new ViewDescriptor())
         .register(ViewGroup.class, new ViewGroupDescriptor())
         .register(Window.class, new WindowDescriptor())
-        .setListener(new DescriptorListener())
+        .setHost(this)
```

Comment:
Curious why this wouldn't be part of the constructor?  It's the only thing not explicitly additive...